src/ralph/data_center/api/views.py

```diff
@@ -55,6 +55,7 @@ class DataCenterAssetViewSet(BaseObjectViewSetMixin, RalphAPIViewSet):
             'ethernet_set',
             queryset=Ethernet.objects.select_related('ipaddress')
         ),
+        'fibrechannelcard_set',  # XXX or 'fibre_channel_card_set'..?
```

Comment:
what about this comment? `fibrechannelcard_set` seems to be working since your tests are passing (otherwise you'll get some error on this prefetch_related)